arcaflow_plugin_stressng/stressng_schema.py

```diff
@@ -563,6 +571,24 @@ class HDDOutput(CommonOutput):
     This is the data structure that holds the results for the HDD stressor
     """
 
+    mbsec_read_rate: typing.Annotated[
+        float,
+        schema.id("mbsec-read-rate"),
+        schema.name("MB/s read rate"),
+    ]
+
+    mbsec_write_rate: typing.Annotated[
+        float,
+        schema.id("mbsec-write-rate"),
+        schema.name("MB/s write rate"),
+    ]
+
+    mbsec_readwrite_combined_rate: typing.Annotated[
+        float,
+        schema.id("mbsec-readwrite-combined-rate"),
+        schema.name("MB/s read-write combined rate"),
+    ]
```

Comment:
It seems like these should be (e.g.) "Read rate (MB/s)" -- why do the units come first?

Reply:
Laziness, I presume. There are a lot of output metrics to define in the schema, so it looks like the initial pass made at these was to lazily convert the key name into a descriptive name.